src/js/edu-benefits/5490/config/form.js

```diff
@@ -145,39 +147,75 @@ const formConfig = {
           path: 'benefits/eligibility',
           initialData: {},
           uiSchema: {
+            'view:benefitsDisclaimerChild': {
+              'ui:description': benefitsDisclaimerChild,
+              'ui:options': {
+                hideIf: form => _.get('relationship', form) !== 'child'
+              }
+            },
+            'view:benefitsDisclaimerSpouse': {
+              'ui:description': benefitsDisclaimerSpouse,
+              'ui:options': {
+                hideIf: form => _.get('relationship', form) !== 'spouse'
+              }
+            },
             benefit: {
               'ui:widget': 'radio',
               'ui:title': 'Select the benefit that is the best match for you:',
               'ui:options': {
                 labels: benefitsLabels,
-                nestedContent: {
-                  chapter33: Chapter33Warning,
-                  chapter35: Chapter35Warning
+                updateSchema: (data, form, schema) => {
+                  const relationship = _.get('relationship', form);
+                  const nestedContent = {
+                    chapter33: benefitSelectionWarning('chapter33', relationship),
+                    chapter35: benefitSelectionWarning('chapter35', relationship),
+                  };
+                  const uiOptions = _.get('benefitSelection.uiSchema.benefit.ui:options', form);
+                  uiOptions.nestedContent = nestedContent;
```

Comment:
I'm a little confused how the nestedContent actually gets rendered. Assuming `chapter33` is selected, what nestedContent will look like is:
```
{
  chapter 33: (<div>{warningText}</div>),
  chapter35: (<div>''</div>)
}
```
And then how does the form render what's actually in that object?